src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -193,6 +201,111 @@ private List<string> ResolveByExternalReferencePackages(List<string> uidList, Co
             return list;
         }
 
+        private async Task<List<string>> ResolveByXRefServiceAsync(List<string> uidList, ConcurrentDictionary<string, XRefSpec> externalXRefSpec)
```

Comment:
ok, then i need look how to do it

---
In reply to: [127627362](https://github.com/dotnet/docfx/pull/1837#discussion_r127627362) [](ancestors = 127627362)